proj/src/main/webapp/signup.js

```diff
@@ -8,30 +8,50 @@ function initSignUp(){
     location.getLocation();
 }
 
+// if(btnLogin){
+//         btnLogin.addEventListener("click", e => {
+//             const emailVal = txtEmail.value;
+//             const passVal = txtPassword.value;
+
+//             // Initialize auth object
+//             const auth = firebase.auth();
+
+//             const promise = auth.signInWithEmailAndPassword(emailVal, passVal).then(function(){
+//                 window.location.replace("chat.html");
+//             });
+//             promise.catch(e => alert(e.message));
+//         });
+//     }
+
```

Comment:
Can this be removed?

Reply:
I think so; I wasn't sure if the original author was intending to come back to this

Reply:
since it is the loginBtn, I think this is in the welcome.js file. If it was already in this file, I think I might have done that by mistake. Github shows it as an addition, though.

Reply:
gotcha i'll remove this